app/src/main/java/net/aiscope/gdd_app/model/SampleModels.kt

```diff
@@ -56,6 +53,54 @@ data class MicroscopeQuality(
     val magnification: Int
 )
 
+data class Images(
```

Comment:
So this `Images` class consists of a list of captures. Wouldn't it be better to name it something like `Captures` or `CaptureList`? It confused me a bit when reading code in other classes

Reply:
I agree, renamed in b66dff67